cli/src/commands/split.rs

```diff
@@ -70,6 +71,12 @@ pub(crate) struct SplitArgs {
         add = ArgValueCandidates::new(complete::mutable_revisions)
     )]
     revision: RevisionArg,
+    /// The change description to use (don't open editor)
+    ///
+    /// The description is used for the split commit. The source
```

Comment:
"the split commit" is not clear to me. It's pretty clear once you're read the next sentence, but I like the phrasing you used in the commit description and in the changelog (i.e. "the commit with the selected changes")